.github/workflows/build-soroban-dev.yml

```diff
@@ -35,10 +35,10 @@ jobs:
     with:
       arch: amd64
       tag: soroban-dev-amd64
-      core_ref: c0ad35aa19297e112d71fcc5755458495f99a237
+      core_ref: 3a201d2b5ecee34d6dcc0466633783f27b7fa97c
       core_configure_flags: --disable-tests --enable-next-protocol-version-unsafe-for-production
-      go_ref: soroban-v0.0.4
-      soroban_tools_ref: v0.4.0
+      go_ref: soroban-xdr-next
+      soroban_tools_ref: auth-next2
```

Comment:
shouldn't this ref be to the release tag on repo, like `v0.5.0` rather than this dev branch?

Reply:
Probably, but I want to test with the branch before it is merged. The release isn't tagged yet.

Reply:
ok, can also do equivalent test with the branch locally by running quickstart's Makefile with same parameters to generate same image locally that gh workflow would have pushed as pr tagged image:

```
# build stellar/quickstart:dev image first with server versions
quickstart$ make CORE_REF=3a201d2b5ecee34d6dcc0466633783f27b7fa97c CORE_CONFIGURE_FLAGS="--disable-tests --enable-next-protocol-version-unsafe-for-production" SOROBAN_TOOLS_REF=auth-next2 GO_REF=soroban-xdr-next build

# run the new stellar/quickstart:dev image
$ docker run --rm -it -p 8000:8000 --name stellar stellar/quickstart:dev --logs --standalone --enable-core-artificially-accelerate-time-for-testing --enable-soroban-rpc --enable-horizon-captive-core

```

Reply:
> I want to test with the branch before it is merged. The release isn't tagged yet.

This is a neat idea. Could we add this to the release process as a formal step we do each time? It'd help us identify any e2e issues earlier.

Reply:
@leighmcculloch , does line 40 on release checklist for 'Smoke Tests' help on this, it would entail [running system-test ](https://github.com/stellar/system-test#runing-tests-using-the-docker-image) which internally performs same build of components at source tag/ref versions and then runs the image and some automated e2e tests against it. additional manual e2e tests could be performed against that rpc url, as system-test only has a few basic e2e tests implemented so far.

Reply:
Yep, https://github.com/stellar/quickstart/pull/421#discussion_r1102069725 would definitely be better. And I should figure out why the quickstart build fails on my local machine at some point.

Reply:
@paulbellamy , I think you can close this pr correct? The preview 7 updates for versions are on this PR - https://github.com/stellar/quickstart/pull/424